go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/grafana/nethax
 go 1.24.2
 
 require (
-	github.com/goccy/go-yaml v1.17.1
 	github.com/spf13/cobra v1.8.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
🤔 actually we should be using the previous package (`goccy`) as this one is [archived](https://github.com/go-yaml/yaml) and thus deprecated. Nevertheless, leave it as it is and we can fix that in a subsequent PR.